internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -450,8 +482,12 @@ func Builder() *cobra.Command {
 		Short: "Create and access an Atlas Cluster.",
 		Long:  "This command creates a cluster, adds your public IP to the atlas access list and creates a db user to access your MongoDB instance.",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if err := opts.ValidateProjectID(); err != nil {
```

Comment:
this may not be enough check, you need to check the three values (project, public and private key) since you no longer do a "validate credentials"

It may not be the common case but all these are optional when doing `mongocli config` so you need to check that all have been set when running this command

Reply:
I moved the check of the project ID where it was before, I believe we should open the browser only if the API keys are not set.

Reply:
agree